curator/Dockerfile

```diff
@@ -1,7 +1,7 @@
 ### This is a generated file from Dockerfile.in ###
 
-#@follow_tag(registry.redhat.io/ubi8/python-36:latest)
-FROM registry.ci.openshift.org/ocp/builder:ubi8.python.36
+#@follow_tag(registry.redhat.io/ubi9/python-39:latest)
+FROM registry.redhat.io/ubi9/python-39:latest
```

Comment:
Since we will need to mirror this image into our upstream CI as `registry.ci.openshift.org/ocp`, you can already start adding the supplemental mirroring here https://github.com/openshift/release/blob/44003930b882788415c17c8b56d24c235c02ca02/core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ocp#L12